src/decompiler.cc

```diff
@@ -67,21 +73,16 @@ struct Decompiler {
   }
 
   void IndentValue(Value &val, size_t amount, string_view first_indent) {
-    WABT_TRACE_ARGS(IndentValue, "\"" PRIstringview "\" - %d",
-                    WABT_PRINTF_STRING_VIEW_ARG(val.v[0]), val.v.size());
     auto indent = Indent(amount);
-    for (auto& s : val.v) {
-      auto is = (&s != &val.v[0] || first_indent.empty())
-                ? string_view(indent)
-                : first_indent;
-      s.insert(0, is.data(), is.size());
+    for (auto& stat : val.v) {
+      auto is = (&stat != &val.v[0] || first_indent.empty())
+                    ? string_view(indent)
+                    : first_indent;
```

Comment:
Is this what clang-format says is correct?

Reply:
Yup, this has been clang-formatted